django_project/lesson/views/section.py

```diff
@@ -137,6 +138,52 @@ def get_queryset(self):
             raise Http404('Sorry! We could not find your section!')
 
 
+class SectionDetailView(SectionMixin, DetailView):
```

Comment:
Why are you adding back this block of code?
It has been removed, it's the Worksheet-list view

Reply:
I didnt know whether it was staying or going